_includes/components/department-budgets/SmallMultiplesSection/index.html

```diff
@@ -4,6 +4,50 @@ <h3 class="Section-title">{{ include.title }}</h3>
     <p>{{ include.description }}</p>
   </div>
 
+  {% if include.dataset or include.guide or include.pdf or include.excel %}
+    <div class="LinksList u-marginBottom10 u-marginLeft30">
+
+      {% if include.dataset %}
+        <div>
+          <a href="{{ include.dataset }}" class="LinksList-item u-textDecorationNone">
+            <span class="LinksList-icon">{% include components/universal/Icon/index.html type="dataset" %}</span>
+            <div class="LinksList-title">
+              <span>Source:&nbsp;</span> 
+              <span class="u-textDecorationUnderline">{{ include.source_type }} {{ include.year }}</span>
+            </div>
+          </a>
+        </div>
+      {% endif %}
+
+      {% if include.guide %}
+        <div>
+          <a href="{{ include.guide }}" class="LinksList-item">
+            <span class="LinksList-icon">{% include components/universal/Icon/index.html type="guide" %}</span>
+            <span class="LinksList-title">Dataset Guide for {{ include.source_type }}</span>
+          </a>
+        </div>
+      {% endif %}
+
+      {% if include.pdf %}
+        <div>
+          <a href="{{ include.pdf }}" class="LinksList-item" target="_blank" rel="noopener noreferrer">
+            <span class="LinksList-icon">{% include components/universal/Icon/index.html type="download" %}</span>
+            <span class="LinksList-title">Learn more about these programmes in the {{ source_type }} as PDF</span>
```

Comment:
I'm trying to wrap my head around when this Jekyll-generated HTML is used, and when the JS-generated HTML is used? What am I missing?

Reply:
Oh, is the HTML the new set of charts, but the JSX is the primary expenditure chart?

Reply:
@longhotsummer correct. Because you need to be able to toggle between with inflation and not.

Reply:
Both the initial programmes chart and expenditure chart data requires a lot of normalisation that I couldn't do in Jekyll. The new chart data that I get is more semantic.

Reply:
Got it. When we migrate that chart to the new chart system, I think we should do all the wording in plain HTML -- that way we can eliminate the duplicate code.